org.osgi.service.servlet/src/org/osgi/service/servlet/whiteboard/HttpWhiteboardConstants.java

```diff
@@ -576,24 +562,19 @@ private HttpWhiteboardConstants() {
 	 * this filter.
 	 *
 	 * @see #HTTP_SERVICE_CONTEXT_PROPERTY
-	 * @since 1.1
 	 */
 	public static final String	HTTP_SERVICE_CONTEXT_FILTER							= "("
 			+ HTTP_SERVICE_CONTEXT_PROPERTY + "=*)";
 
 	/**
 	 * The name of the implementation capability for the Http Whiteboard
 	 * specification
-	 * 
-	 * @since 1.1
 	 */
 	public static final String	HTTP_WHITEBOARD_IMPLEMENTATION						= "osgi.http";
 
 	/**
 	 * The version of the implementation capability for the Http Whiteboard
 	 * specification
-	 * 
-	 * @since 1.1
 	 */
-	public static final String	HTTP_WHITEBOARD_SPECIFICATION_VERSION				= "1.1";
+	public static final String	HTTP_WHITEBOARD_SPECIFICATION_VERSION				= "2";
```

Comment:
```suggestion
	public static final String	HTTP_WHITEBOARD_SPECIFICATION_VERSION				= "2.0";
```

Needs to be major.minor.